lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1337,19 +1337,19 @@ export class UserAgentApplication {
       }
 
       if (accessTokenCacheItem != null) {
-          let expired = Number(accessTokenCacheItem.value.expiresIn);
+          const expired = Number(accessTokenCacheItem.value.expiresIn);
           // If expiration is within offset, it will force renew
           const offset = this.config.system.tokenRenewalOffsetSeconds || 300;
           if (expired && (expired > TimeUtils.now() + offset)) {
-              let idTokenObj = new IdToken(accessTokenCacheItem.value.idToken);
+              const idTokenObj = new IdToken(accessTokenCacheItem.value.idToken);
               if (!account) {
                   account = this.getAccount();
                   if (!account) {
                       throw AuthError.createUnexpectedError("Account should not be null here.");
                   }
               }
               const aState = this.getAccountState(serverAuthenticationRequest.state);
-              let response : AuthResponse = {
+              const response : AuthResponse = {
```

Comment:
We can't make this const, in setResponseIdToken we modify the idToken field.

Reply:
see previous response